shared-module/displayio/ColorConverter.c

```diff
@@ -161,6 +162,9 @@ void displayio_colorconverter_convert(displayio_colorconverter_t *self, const _d
         }
         output_color->pixel = packed;
         output_color->opaque = true;
+        if (self->transparent_color == pixel) {
+            output_color->opaque = false;
+        }
```

Comment:
Do this check once before `if (self->dither){` so that the pixel value hasn't been randomly changed before checking. Once you set opaque to false you'll also want to return. Nothing else needs to run.

Reply:
Thanks for the review! I saw you suggested to change the type to an `int` but then when I get to this comparison C is not happy that I am trying to compare an `int` and `int32`. 😞  That was my original approach. Any ideas?

(Also the API change sounds good I can do that.)

Reply:
I expect it's complaining about signedness. You can cast the int to an unsigned by doing `(uint32_t) int_variable`. It won't cause problems as long as it's >=0.